llvm/lib/Target/DirectX/DXILShaderFlags.cpp

```diff
@@ -62,27 +73,65 @@ static void updateFunctionFlags(ComputedShaderFlags &CSF, const Instruction &I,
     }
     }
   }
+  // Handle call instructions
+  if (auto *CI = dyn_cast<CallInst>(&I)) {
+    const Function *CF = CI->getCalledFunction();
+    // Merge-in shader flags mask of the called function in the current module
+    if (FunctionFlags.contains(CF)) {
+      CSF.merge(FunctionFlags[CF]);
+    }
```

Comment:
nit:
```suggestion
    if (FunctionFlags.contains(CF))
      CSF.merge(FunctionFlags[CF]);
```
see: https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements